spec/controllers/api/v1/short_form_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe Api::V1::ShortFormController, type: :controller do
   login_admin
 
-  listing_id = 'a0W0P00000GbyuQ' # LAP Test Listing - Yellow Acres
+  listing_id = 'a0W0P00000F8YG4UAN' # LAP Test Listing - Yellow Acres
```

Comment:
looks like this is the wrong listing. Should be yellow acres lease-up listing, but you've updated it to automated test listing, our pre-lottery test listing